assets/sass/components/key-metrics/_googlesitekit-km-selection-panel.scss

```diff
@@ -91,6 +91,10 @@
 
 		.googlesitekit-error-text p {
 			margin: 0;
+
+			@media (max-width: $bp-xsmallOnly) {
+				margin: 1em 0;
+			}
```

Comment:
Even though I proposed a max-width of 450px here, now that I come to think of it, I think it might be worth taking a mobile-first approach by only applying `margin: 0` here only when the `min-width` is `$bp-tablet`. `margin: 1em 0;` is applied by default to this element. What do you think?

Reply:
Let's remove this block entirely as `p` has this margin by default.